client/templates/reporting/standard-reports/date-search.njk

```diff
@@ -30,26 +30,13 @@
   <h1 class="govuk-heading-l">{{ title }}</h1>
 
   <form action="{{ reportUrl }}" method="POST" id="dateSearchForm">
-    {{ datePicker({
-      id: "dateFrom",
-      label: {
-        text: "Date from"
-      },
-      hint: "Use dd/mm/yyyy format.",
-      dateValue: tmpBody.dateFrom,
-      dateError: dateFromError
-    }) }}
-
-    {{ datePicker({
-      id: "dateTo",
-      label: {
-        text: "Date to"
-      },
-      hint: "Use dd/mm/yyyy format.",
-      dateMin: tmpBody.dateFrom,
-      dateValue: tmpBody.dateTo,
-      dateError: dateToError
-    }) }}
+    
+    {# we can make this a list of reports if any other uses the same date range selector as reasonable adjustments #}
```

Comment:
It'd be nice to abstract this out to the definitions file, rather than have a list in here.

Reply:
makes sense, I have added a new search type that will handle the different case.